Cargo.toml

```diff
@@ -37,3 +37,4 @@ gaggle = ["nng"]
 
 [dev-dependencies]
 httpmock = "0.3"
+mockito = "0.25"
```

Comment:
@LionsAd This adds more compilation time when building the tests, but currently seems to be the only way to test a redirect -- each mock server runs on a different port. The author of httpmock said he'd explore adding the ability to run multiple servers on different ports, at which time I'll remove mockito again.